src/GraphQL/Types/TimeSpanMillisecondsGraphType.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using GraphQL.Language.AST;
+
+namespace GraphQL.Types
+{
+    public class TimeSpanMillisecondsGraphType : ScalarGraphType
+    {
+        public TimeSpanMillisecondsGraphType()
+        {
+            Name = "TimeSpan";
```

Comment:
These `TimeSpan` types should have unique names.  Otherwise you won't be able to use both in the same schema (one would override the other).

Reply:
Yes of course, I missed that. I've now called the types `Seconds` and `Milliseconds` unless you have a better name. Perhaps `TimeInSeconds` and `TimeInMilliseconds`?

If there are no more suggestions, should I start fixing and writing new tests?